docs/development_environment.mdx

```diff
@@ -65,7 +65,9 @@ It is also possible to debug a remote Home Assistance instance (e.g., production
 
 _You only need these instructions if you do not want to use devcontainers._
 
-It is also possible to set up a more traditional development environment. See the section below for your operating system. Make sure your Python version is 3.12.
+It is also possible to set up a more traditional development environment. See the section below for your operating system.
+
+You currently need to use Python version 3.13 to create the virtual environment. This is not yet common in distributions, so you may want to use something like pyenv.
```

Comment:
I think we should just keep changing the version number. The addition on the distribution is becoming more odd over time, as some might have it, others won't.